vanilla-tests/src/main/java/fixtures/modelflattening/AutoRestResourceFlatteningTestService.java

```diff
@@ -366,7 +366,7 @@ public void putWrappedArray(List<WrappedProduct> resourceArray) {
             return Mono.error(new IllegalArgumentException("Parameter this.getHost() is required and cannot be null."));
         }
         if (resourceDictionary != null) {
-            resourceDictionary.values().forEach(e -> e.validate());
+            resourceDictionary.values().forEach(e -> {if (e != null) {e.validate();}});
```

Comment:
nit: space after { and before }

Reply:
Fixed. checkstyle actually allows code block in single line.